styles/main.css

```diff
@@ -28,8 +46,54 @@ body {
   margin-right: 5%;
 }
 
+#app-bar-desktop {
+  display: none;
+}
+
+.tags ul {
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.about-me ul {
+  justify-content: center;
+}
+
+.recent-work .card ul {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+#app-bar-desktop ul {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 0 30px;
+}
+
+.tags ul a {
+  color: #3a4a42;
+  font-family: Inter, serif;
+  font-size: 0.8rem;
+  font-weight: 600;
+}
```

Comment:
- [ ] [_OPTIONAL_] Great job so far 👍, to make this perfect please add some space with the individual icons and give the m a round curve so that it will completely resemble the template.

### Your implementation
![image](https://user-images.githubusercontent.com/59416136/154565399-f7ee987b-9460-4678-b20d-f6b63847514f.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/154565454-c22a7743-dbf8-44d3-82d9-6509d12f02e2.png)

Reply:
Thank you @Wes-Isaac, ill work on the suggested changes.